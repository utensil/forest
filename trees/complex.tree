\texdef{Definition 1}{Complex Numbers}{
A complex number $z$ is a number of the form $a + bi$ where $a$ and $b$ are real numbers and $i$ is the imaginary unit defined by $$i^2 = -1$$
}

\begin{enumerate}
\item Properties of complex numbers:
  \begin{itemize}
  \item Addition: $(a + bi) + (c + di) = (a + c) + (b + d)i$
  \item Multiplication: $(a + bi)(c + di) = (ac - bd) + (ad + bc)i$
  \ii The conjugate of $a + bi$ is $a - bi$
  \end{itemize}
\item Important formulas:
  \begin{enumerate}
  \item Euler's formula: $$e^{ix} = \cos x + i\sin x$$
  \item De Moivre's formula: $$(\cos \theta + i\sin \theta)^n = \cos(n\theta) + i\sin(n\theta)$$
  \end{enumerate}
\end{enumerate}

\texnote{Historical Note}{Discovery of Complex Numbers}{
The concept of complex numbers emerged from attempts to solve cubic equations in the 16th century.
\emph{Gerolamo Cardano} was among the first to systematically use these "impossible" numbers.
}

\minitex{
The term "imaginary number" was coined by \emph{Ren√© Descartes} in 1637, who used it as a derogatory term.
The modern notation $i$ was introduced by \emph{Leonhard Euler} in 1777.
}
