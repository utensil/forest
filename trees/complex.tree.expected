\refdef{Definition 1}{Complex Numbers}{

\p{
A complex number #{z} is a number of the form #{a + bi} where #{a} and #{b} are real numbers and #{i} is the imaginary unit defined by ##{i^2 = -1}
}

\ol{
\li{Properties of complex numbers:}
  \ul{
  \li{Addition: #{(a + bi) + (c + di) = (a + c) + (b + d)i}}
  \li{Multiplication: #{(a + bi)(c + di) = (ac - bd) + (ad + bc)i}}
  \li{The conjugate of #{a + bi} is #{a - bi}}
  }
\li{Important formulas:}
  \ol{
  \li{Euler's formula: ##{e^{ix} = \cos x + i\sin x}}
  \li{De Moivre's formula: ##{(\cos \theta + i\sin \theta)^n = \cos(n\theta) + i\sin(n\theta)}}
  }
}

\refnote{Historical Note}{Discovery of Complex Numbers}{

\p{
The concept of complex numbers emerged from attempts to solve cubic equations in the 16th century.
\em{Gerolamo Cardano} was among the first to systematically use these "impossible" numbers.
}

{

\p{
The term "imaginary number" was coined by \em{Ren√© Descartes} in 1637, who used it as a derogatory term.
The modern notation #{i} was introduced by \em{Leonhard Euler} in 1777.
}

}
